python/cuml/tests/test_random_forest.py

```diff
@@ -210,8 +210,8 @@ def test_default_parameters():
     assert clf_params["max_features"] == "sqrt"
 
     # Different default split_criterion
-    assert reg_params["split_criterion"] == 2
-    assert clf_params["split_criterion"] == 0
+    assert reg_params["split_criterion"] == "mse"
```

Comment:
Before we used the opaque numeric versions as the default values. To ease sklearn <-> cuml roundtrip (where we have a test that hyperparameters haven't changed) I switched to using the string versions as defaults. This results in no functionality change for the user, `"mse"` and `2` both mean the same thing in this API.